corrector.rb

```diff
@@ -0,0 +1,9 @@
+class Corrector
+  def correct_name(name)
+    if name[0].upcase and name.length > 10
+      name.chr.upcase + name[1..9]
+    else
+      name.chr.upcase + name[1..name.length]
+    end
+  end
+end
```

Comment:
- [ ] Instead of using 2 methods you could use the `.capitalize!` method 😄
- [ ] Using this method you can return something like this instead:
    - `name.capitalize![0,10]` in either case, and there is no need of using an if condition because if the word length is smaller than 10 it will return the spaces that exist, kindly try this in your terminal 😄

Reply:
Done as suggested :sweat_smile: Thank you for the review. Your solution is cool and simple as well